docs/build/dev-workflow/personas.md

```diff
@@ -11,13 +11,13 @@ The aim is to provide an overview of the main technical user personas, how they
 
 ## Polymer, Ethereum's IBC hub
 
-Polymer is bringing IBC to Ethereum rollups. [IBC](../../learn/concepts/ibc/ibc.md) or the Inter-Blockchain communication protocol is a layered protocol that does a good job of separating the concerns of application developers, and core developers (consisting of transport and state layers).
+Polymer is bringing IBC to Ethereum rollups. [IBC](../../learn/concepts/ibc/ibc.md) or the _Inter-Blockchain communication protocol_ is a layered protocol that does a good job of separating the concerns of application developers, and core developers (consisting of transport and state layers).
```

Comment:
[Suggestion]:

Polymer is bringing _Inter-Blockchain communication protocol_ (IBC) to Ethereum rollups. [IBC](../../learn/concepts/ibc/ibc.md) is a blockchain interoperability protocol enabling secure, permissionless, and feature-rich cross-chain interactions. IBC is a layered protocol that that has clear separation between application, transport and state layers. This creates a great developer experience because it separates the concerns of application developers from those of core developers (consisting of transport and state layers).